app/views/layouts/_navbar.html.erb

```diff
@@ -1,5 +1,5 @@
 <nav>
-  <button id='back-button'><%= link_to "Back", root_path %></button>
+  <button id='back-button'><%= link_to "Home", root_path %></button>
   <h1>Blog App</h1>
   <%= button_to "Sign Out", destroy_user_session_path, method: :delete, class: "sign-out" %>
 </nav>
```

Comment:
- [ ] [OPTIONAL] It would be better If you could show the sign-in page by default for unauthenticated users instead of displaying the home page as I had to manually write the `/users/sign_in` path to register and signin into your app because there's no way to go to the login/registration page. Also, the signout button isn't working properly as shown below so I'd suggest you fix this.
  ![firefox_OvC573jUuZ](https://github.com/tan12082001/project-blog-app/assets/104968236/2dcd6e70-6b01-4201-9f95-4ef0311c4cd8)